tests/bsim/bluetooth/host/misc/disable/src/main_disable.c

```diff
@@ -35,6 +35,66 @@ static void test_disable_main(void)
 	PASS("Disable test passed\n");
 }
 
+static void test_disable_set_default_id(void)
+{
+	for (int i = 0; i < NUM_ITERATIONS; i++) {
+
+		/* FIXME: Temporary workaround to get around a bug in the controller
+		 * Open PR: https://github.com/zephyrproject-rtos/zephyr/pull/73342
+		 */
+		k_sleep(K_MSEC(1));
```

Comment:
1. Describe the symptom of the bug so it can be reproducible. As I see, it is the POWER_CLOCK_IRQn that is not executed if a test is developed without any cpu_idle() call.
2. The workaround to sleep is only needed once, not needed in a loop. See the original suggestion here: https://github.com/zephyrproject-rtos/zephyr/pull/72680/files#r1599770565

Reply:
Updated the description. Hopefully it can be removed soon. The linked PR should give enough information